web/views/manifestation.py

```diff
@@ -0,0 +1,38 @@
+from flask import request, Blueprint
+from flask_restful import Resource, Api
+
+from coalaip_bigchaindb.plugin import Plugin
+from coalaip import CoalaIp
+
+
+coalaip = CoalaIp(Plugin('http://localhost:9984/api/v1'))
```

Comment:
For now I'm just going to leave this as `'http://{}:{}/api/v1'`, we can figure out the API path later with the driver.